test/run

```diff
@@ -219,14 +219,26 @@ expect_file_contains() {
     local file="$1"
     local string="$2"
 
-    if [ ! -f "$file" ]; then
+    if [ ! -e "$file" ]; then
         test_failed "$file not found"
     fi
     if ! grep -q "$string" "$file"; then
         test_failed "File $file does not contain: $string. Actual content: $(cat $file)"
     fi
 }
 
+expect_file_not_contains() {
+    local file="$1"
+    local string="$2"
+
+    if [ ! -e "$file" ]; then
+        test_failed "$file not found"
+    fi
+    if grep -q "$string" "$file"; then
```

Comment:
```suggestion
    if fgrep -q "$string" "$file"; then
```

Reply:
I actually would have preferred `fgrep`, but the complementary `expect_file_contains()` uses `grep`, so I figured you intended to support/require regexps.

Reply:
Right. I didn't react to the `grep` in 3eb2483f66fc482df3bf31616d35daedc5ac49cc and there was no intention to support regexes.

Reply:
[Fixed.](https://github.com/ccache/ccache/pull/596/commits/163bf67ab4d279f5b008a9d2ca24f0caaab27a5c)